REVIEW_GUIDELINES.md

```diff
@@ -44,8 +45,27 @@ labels:
   heritage: {{ .Release.Service }}
 ```
 
+If a chart has multiple components, a `component` label should be added (e. g. `component: server`). The resource name should get the component as suffix (e. g. `name: {{ template "myapp.fullname" . }}-server`).
+
 Note that templates have to be namespaced. With Helm 2.7+, `helm create` does this out-of-the-box. The `app` label should use the `name` template, not `fullname` as is still the case with older charts.
 
+### Deployments and StatefulSets selectors
```

Comment:
We should probably add `DaemonSet` as well.

Reply:
Thanks, I realized after writing this that DS were affected as well.